react-commenting.js

```diff
@@ -66,18 +66,18 @@ export default Board
 import React, { Component } from 'react'
 
 class Dice extends Component{
-  // 7)
+  // 7) here it seems we are in the Dice component which extends our original React parent Component.
   render(){
-    // 8)
+    // 8) this will then render it's assined task back to the parent component so it can be displayed, perhaps on a webpage or a browser, which in this case looks like this Component is in charge of rolling the dice, and displays a current picture of the results eg. "current number is 6".
     const { currentPic, roll } = this.props
     return(
       <div id="dice-container">
         <h2>Click to Roll</h2>
         <img
           id="dice-img"
-          // 9)
+          // 9)the id tags are identifiers that allow other components, such as the css component, to call back on a specific component we'd like to customize and add artwork to it.  all of these are able to be displayed thanks to the <div> tag, which is a tag that lets HTML recieve and display JavaScript functions.
           alt="images of dice"
-          // 10)
+          // 10)and last but definitely not least, we finish giving the functions to the child component, and export all of this, returning and connecting the Child Component back to the original Parent Component.
```

Comment:
good work on this file! however, we are looking for the purpose of the code _below_ the number.